sspl_test/coverage/stop_sspl_coverage.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/sh
+
+# iofdir='/opt/sumedh/withoutpyi/cortx-sspl/low-level/framework'
+iofdir='/opt/seagate/cortx/sspl/low-level/framework'
+
+echo "Generating the coverage report.."
+consul kv put sspl/config/SYSTEM_INFORMATION/log_level DEBUG
```

Comment:
why changing log level here ?

Reply:
It triggers set_log_level() method in which we are injecting code to stop and generate coverage report.

Reply:
We can also trigger it using `kill -10 pid` if that seems appropriate

Reply:
enabling debug level will unecessarily bulk up the logs, when we dont need really. 

"kill -10" i.e SIGUSR1 is again getting used to signal log level change only, so same handler will be invoked, but yes no log level will be changed, seems fine then, can do that.